dipy/align/tests/test_imwarp.py

```diff
@@ -586,36 +612,21 @@ def test_ssd_3d_demons():
     mapping = optimizer.optimize(static, moving, None)
     m = optimizer.get_map()
     assert_equal(mapping, m)
-    energy_profile = subsample_profile(
-        optimizer.full_energy_profile, 10)
-    print(energy_profile)
 
-    if USING_VC_SSE2:
-        expected_profile = \
-            np.array([312.22706987, 154.65556884, 53.88543188, 9.11484007,
-                      36.46592407, 13.20522299, 48.65663399, 14.91579802,
-                      49.82954704, 14.92646254])
-    elif USING_GCC_SSE2:
-        expected_profile = \
-            np.array([312.22706987, 154.65556885, 53.88455398, 9.11770682,
-                      36.48642824, 13.21706748, 48.67710635, 14.91782047,
-                      49.84142899, 14.92531294])
+    warped = mapping.transform(moving)
+    starting_energy = np.sum((static - moving)**2)
+    final_energy = np.sum((static - warped)**2)
+    reduced = 1.0 - final_energy/starting_energy
 
-    assert_array_almost_equal(energy_profile, expected_profile, decimal=4)
+    assert(reduced > 0.9)
 
 
-@npt.dec.skipif(NO_SSE2)
 def test_ssd_3d_gauss_newton():
-    r'''
-    Register a stack of circles ('cylinder') before and after warping them with
-    a synthetic diffeomorphism. This test is intended to detect regressions
-    only: we saved the energy profile (the sequence of energy values at each
-    iteration) of a working version of SSD in 3D using the Gauss-Newton step,
-    and this test checks that the current energy profile matches the saved
-    one. The validation of the "working version" was
-    done by registering the 18 manually annotated T1 brain MRI database IBSR
-    with each other and computing the jaccard index for all 31 common
-    anatomical regions.
+    r''' Registration: test 3D SyN with SSD metric, Gauss-Newton optimizer
```

Comment:
Same as before please remove ```Registration:``` from all tests. You are in the alignment module so these tests are of course about registration.